tsl/src/bgw_policy/job.c

```diff
@@ -391,8 +449,14 @@ policy_compression_execute(int32 job_id, Jsonb *config)
 	if (chunkid != INVALID_CHUNK_ID)
 	{
 		Chunk *chunk = ts_chunk_get_by_id(chunkid, true);
-		tsl_compress_chunk_wrapper(chunk, false);
-
+		if (hypertable_is_distributed(policy_data.hypertable))
+		{
```

Comment:
Nit: I would elide braces for single line statements.

Reply:
I original did, but `make format` did that...

Reply:
I made another attempt with `make format` and now it's:
```
		if (hypertable_is_distributed(policy_data.hypertable))
			policy_invoke_compress_chunk(chunk);
		else
			tsl_compress_chunk_wrapper(chunk, true);
```
which is much better.